cmd/lotus-pcr/main.go

```diff
@@ -884,6 +919,8 @@ func (r *refunder) processTipsetStorageMarketActor(ctx context.Context, tipset *
 		}
 
 		refundValue = types.BigMul(types.NewInt(uint64(recp.GasUsed)), tipset.Blocks()[0].ParentBaseFee)
+	default:
+		return false, messageMethod, types.NewInt(0), nil
```

Comment:
unrelated to blocklist, this was a fix I made yesterday and hadn't pr'd yet. It stops us from returning `true` for message methods we don't know about.